src/coord/config.c

```diff
@@ -53,39 +53,20 @@ CONFIG_GETTER(getClusterTimeout) {
 }
 
 CONFIG_SETTER(setGlobalPass) {
-  SearchClusterConfig *realConfig = getOrCreateRealConfig(config);
-  if (realConfig->globalPass) {
-    rm_free((void *)realConfig->globalPass);
-  }
-  int acrc = AC_GetString(ac, &realConfig->globalPass, NULL, 0);
-  RETURN_STATUS(acrc);
-}
```

Comment:
We do not want to keep this configuration when we run on older Redis versions (that does not have internal secret)?